sormas-api/src/main/java/de/symeda/sormas/api/travelentry/TravelEntryDto.java

```diff
@@ -0,0 +1,426 @@
+package de.symeda.sormas.api.travelentry;
+
+import java.util.Date;
+
+import de.symeda.sormas.api.Disease;
+import de.symeda.sormas.api.caze.CaseReferenceDto;
+import de.symeda.sormas.api.caze.EndOfIsolationReason;
+import de.symeda.sormas.api.caze.QuarantineReason;
+import de.symeda.sormas.api.contact.QuarantineType;
+import de.symeda.sormas.api.disease.DiseaseVariant;
+import de.symeda.sormas.api.infrastructure.PointOfEntryReferenceDto;
+import de.symeda.sormas.api.person.PersonReferenceDto;
+import de.symeda.sormas.api.region.CommunityReferenceDto;
+import de.symeda.sormas.api.region.DistrictReferenceDto;
+import de.symeda.sormas.api.region.RegionReferenceDto;
+import de.symeda.sormas.api.user.UserReferenceDto;
+import de.symeda.sormas.api.utils.EmbeddedPersonalData;
+import de.symeda.sormas.api.utils.Required;
+import de.symeda.sormas.api.utils.SensitiveData;
+import de.symeda.sormas.api.utils.YesNoUnknown;
+import de.symeda.sormas.api.utils.pseudonymization.PseudonymizableDto;
+
+public class TravelEntryDto extends PseudonymizableDto {
+
+	@Required
+	@EmbeddedPersonalData
+	private PersonReferenceDto person;
+	private Date reportDate;
+	private UserReferenceDto reportingUser;
+	private boolean archived;
+	private boolean deleted;
+	private Disease disease;
+	@SensitiveData
+	private String diseaseDetails;
+	private DiseaseVariant diseaseVariant;
+	private RegionReferenceDto responsibleRegion;
+	private DistrictReferenceDto responsibleDistrict;
+	private CommunityReferenceDto responsibleCommunity;
+	private RegionReferenceDto pointOfEntryRegion;
+	private DistrictReferenceDto pointOfEntryDistrict;
+	private PointOfEntryReferenceDto pointOfEntry;
+	@SensitiveData
+	private String pointOfEntryDetails;
+	@EmbeddedPersonalData
+	private CaseReferenceDto resultingCase;
+	private String externalId;
+	private boolean recovered;
+	private boolean vaccinated;
+	private boolean testedNegative;
+	@SensitiveData
+	private String deaContent;
+
+	private QuarantineType quarantine;
+	@SensitiveData
+	private String quarantineTypeDetails;
+	private Date quarantineFrom;
+	private Date quarantineTo;
+	@SensitiveData
+	private String quarantineHelpNeeded;
+	private boolean quarantineOrderedVerbally;
+	private boolean quarantineOrderedOfficialDocument;
+	private Date quarantineOrderedVerballyDate;
+	private Date quarantineOrderedOfficialDocumentDate;
+	private YesNoUnknown quarantineHomePossible;
+	@SensitiveData
+	private String quarantineHomePossibleComment;
+	private YesNoUnknown quarantineHomeSupplyEnsured;
+	@SensitiveData
+	private String quarantineHomeSupplyEnsuredComment;
+	private boolean quarantineExtended;
+	private boolean quarantineReduced;
+	private boolean quarantineOfficialOrderSent;
+	private Date quarantineOfficialOrderSentDate;
+
+	private YesNoUnknown wasInQuarantineBeforeIsolation;
```

Comment:
Isolation fields are not necessary - sorry, I missed those when I refined the issue